Utilities/CodeGeneration/EdFi.Ods.CodeGen/Generators/Resources/PropertyData.cs

```diff
@@ -346,5 +355,52 @@ public static PropertyData CreateNullProperty(ResourceProperty property)
 
             return propertyData;
         }
+
+        private static bool CSharpDefaultHasDomainMeaning(ResourceProperty property)
```

Comment:
Rather than copy this method here, I think we should look at converting this into an extension method on the `PropertyTypeExtensions` class. Also, then, remove the one in the _Entities.cs_ generator class and change code to use the shared version of this function there and here.

I'm thinking of the following signature:
```csharp
public static bool CSharpDefaultHasDomainMeaning(this PropertyType propertyType, string propertyName)
{
    ...
}
```